Gruntfile.js

```diff
@@ -14,7 +14,9 @@ const extract = require('jsxgettext-recursive');
 // use this if you want to recursively match all subfolders:
 // 'test/spec/**/*.js'
 module.exports = function (grunt) {
-  'use strict';
+  'use strict'
```

Comment:
Yeah, looks like this Gruntfile is the exception in this repo. Can we turn it on for just this file?